controllers/project.js

```diff
@@ -0,0 +1,63 @@
+const Sequelize = require('sequelize');
+const express = require('express');
+const passport = require('passport');
+const router = express.Router();
+const db = require('../models');
+const ensureAuthenticated = require('../util/helpers').ensureAuthenticated;
+
+// Route base is '/project'
+
+// PROJECT ROUTES
+/////////////////////////////////////
+router.get('/', ensureAuthenticated, (req, res) => {
+  db.Project.findAll({ where: { user_id: req.user.id } }).then(result => {
+    res.render('project', {
+      projects: result
+    });
+  });
```

Comment:
Does this need to render a page?

Reply:
i think the only other page we're rendering is a project page with all the milestones